src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$nav-color: #00acdc;
+$color-title-card: #060b35;
+$regular-card-text: #616070;
+$nav-indents: 0;
+$position-item-border: 0;
```

Don't need to duplicate variables
```suggestion
$nav-indents: 0;
```